gpflow/config/__config__.py

```diff
@@ -124,12 +139,18 @@ def set_default_jitter(value: float):
     set_config(replace(config(), jitter=value))
 
 
-def set_default_summary_fmt(value: str):
-    formats = tabulate.tabulate_formats + ['notebook', None]
-    if value not in formats:
-        raise ValueError(f"Summary does not support '{value}' format")
+_VALID_POSITIVE_BIJECTORS = {"exp", "softplus"}
```

Comment:
Could you put it either into the function or in the beginning of the file. (I prefer function itself)